pkg/compose/build.go

```diff
@@ -139,6 +139,9 @@ func (s *composeService) ensureImagesExists(ctx context.Context, project *types.
 				project.Services[i].Labels = types.Labels{}
 			}
 			project.Services[i].CustomLabels[api.ImageDigestLabel] = digest
+			if service.Image != "" {
+				project.Services[i].CustomLabels[api.ImageNameLabel] = service.Image
+			}
```

Comment:
Shouldn't this _always_ add it (even if it's blank)?

Reply:
Good catch!